common/presenters/move-to-card-component.js

```diff
@@ -10,7 +10,10 @@ function moveToCardComponent({
   showStatus = true,
   hrefSuffix = '',
 } = {}) {
-  return function item({ id, reference, person, status }) {
+  return function item(
+    { id, reference, person, status },
+    cancelAllocationHref
```

Comment:
So this feels like a similar scenario Alex got into around "actions" within a card. I feel that the use of all the "allocation" specific terminology in here could be made a little more generic as the purpose of the components is to be more content agnostic.

It looks like what we're attempting to do is to say in this instance we want to provide actions, but it doesn't actually look like we're passing them down to the component so I'm not sure the presenter is the best place for this logic.

I've made a further comment about how you could potentially bypass all of these changes to the presenter, similar to how Alex implemented the remove button.